.github/workflows/cd.yml

```diff
@@ -0,0 +1,164 @@
+# This workflow builds the binaries, which are released when a "v*" tag is pushed.
+# These also include builds for 32bit platforms, which are thus tested as a side-effect.
+# This workflow only runs on a push to master and when pushing a version-tag.
+#
+# The Linux builds are performed on a "manylinux2010" container. This container
+# is designed such that that the resulting binary has minimal dependencies on system
+# libraries, and thus works on as many linuxes as possible. It's a thing from the
+# Python world, but generally useful.
+
+name: CD
+
+on:
+  push:
+    tags: [ 'v*' ]
+    branches: [ master, cd ]
```

Comment:
Small but useful hack, if the branch is called "cd" this does run, which is useful when working on the pipeline code ;)